resources/secrets.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: kyma-showcase-secret
+  namespace: default
+  selfLink: /api/v1/namespaces/default/secrets/kyma-showcase-secret
+  managedFields:
+    - manager: kubectl-create
+      operation: Update
+      apiVersion: v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        f:data:
+          .: {}
+          f:REDIS_PASSWORD: {}
+          f:GCP_API_KEY: {}
+          f:GCP_EMAIL: {}
```

Comment:
Do we really need these fields? Look like a combination of parameters injected by the Kubernetes and then copied to the file